egui/src/ui.rs

```diff
@@ -1055,7 +1055,7 @@ impl Ui {
     /// Add extra space before the next widget.
     ///
     /// The direction is dependent on the layout.
-    /// This will be in addition to the [`Spacing::item_spacing`}.
+    /// This will be in addition to the [`crate::style::Spacing::item_spacing`].
```

Comment:
Didn't the old link work properly? If so, shouldn't the CI/`check.sh` catch that?

Reply:
No, it looks as a monospaced text in the brackets